controllers:
  imu_complementary_filter_controller:
    type: imu_filter_controllers/ComplementaryController
    name: "gimbal_imu"
    do_bias_estimation: false
    do_adaptive_gain: true
    gain_acc: 0.0003
